app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    components = command.split()
+    command_, source_file, destination_file = command.split()
+
+    if len(components) == 3 and command_ == "cp":
+        if source_file != destination_file:
```

Comment:
You able to merge this if statements
```suggestion
    if len(components) == 3 and command_ == "cp" and source_file != destination_file:
```